src/main/java/pl/simpleascoding/tutoringplatform/repository/RoleRepository.java

```diff
@@ -2,7 +2,10 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 import pl.simpleascoding.tutoringplatform.domain.user.Role;
+import pl.simpleascoding.tutoringplatform.domain.user.RoleType;
+
+import java.util.List;
 
 public interface RoleRepository extends JpaRepository<Role, Long> {
-    //todo
+    List<Role> findRoleByRoleType(RoleType roleType);
```

Comment:
Please rename this method to `findRolesByRoleType`, as it's searching for multiple Role objects

Reply:
We should rework our roles system to allow searching for users directly in `UserRepository`, but for now it's a valid solution.